google-cloud-clients/google-cloud-contrib/.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "cloudasset",
```

Comment:
How come the name here is `cloudasset` whereas the directory is `contrib`?

Reply:
oops, that file should not have been included